go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	k8s.io/client-go v0.21.3
 	k8s.io/code-generator v0.21.3
 	k8s.io/component-base v0.21.3
+	k8s.io/klog v1.0.0
```

Comment:
The project already imports `klog/v2`, so we should avoid importing `klog`.

Reply:
Done.